nodejs/week1/homework/src/backend/routes/meal.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("../data/meals.json");
+const reviews = require("../data/reviews.json");
+
+router.get("/meal", (request, response) => {
```

Comment:
This `"/meal"` path actually needs to be `"/"` because you're already setting the route to `"/meal"` in app.js. These compound to "/meal/meal" which is not what you want.  :)